src/DocumentFormat.OpenXml/Spreadsheet/CellValue.cs

```diff
@@ -116,7 +116,26 @@ public bool TryGetDecimal(out decimal value)
         /// <param name="value">The result if successful.</param>
         /// <returns>Success or failure</returns>
         public bool TryGetBoolean(out bool value)
-            => bool.TryParse(InnerText, out value);
+            => bool.TryParse(InnerText, out value) || TryGetBooleanFromInteger(out value);
+
+        private bool TryGetBooleanFromInteger(out bool value)
```

Comment:
This method can be `static`.

Reply:
It was using an instance member `InnerText`